CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased Phase 2
```

Comment:
This isn't your fault, but the CHANGELOG for this branch seems like it got messed up. Things like this shouldn't be nested under release 10, since they weren't released yet: 

https://github.com/nasa/cumulus/pull/2683/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR119

Reply:
So we need to fix that on feature/rds-phase-2 before we can merge this